enterprise-suite/values.yaml

```diff
@@ -28,6 +28,11 @@ imagePullPolicy: IfNotPresent
 prometheusDomain: prometheus.io
 # minikube debug resources
 minikube: true
+# commercial image credentials - username and password are required
+imageCredentials:
+  registry: lightbend-docker-commercial-registry.bintray.io
+  username: setme
+  password: setme
```

Comment:
Is this encrypted or something?  This seems dangerous otherwise but maybe I'm missing something.

Reply:
What should be encrypted?